conda_smithy/feedstock_content/ci_support/upload_or_check_non_existence.py

```diff
@@ -44,7 +44,7 @@ def upload(cli, meta, owner, channels):
     try:
         with open('binstar.token', 'w') as fh:
             fh.write(cli.token)
-        subprocess.check_call(['binstar', '--quiet', '-t', 'binstar.token',
+        subprocess.check_call(['anaconda-client', '--quiet', '-t', 'binstar.token',
```

Comment:
I'm saving up a few changes and will submit a PR once the Obvious-ci feedstock is doing its business.